apps/condo/domains/property/schema/CheckPropertyWithAddressExistService.js

```diff
@@ -5,13 +5,19 @@
 const { GQLCustomSchema } = require('@core/keystone/schema')
 const access = require('@condo/domains/property/access/CheckPropertyWithAddressExistService')
 const { Property } = require('@condo/domains/property/utils/serverSchema')
+const get = require('lodash/get')
+const { jsonAddressMetaValidator } = require('../utils/validation.utils')
 
+const NO_OBJECT_MESSAGE = 'No object specified!'
+const META_INCORRECT_JSON_MESSAGE = 'AddressMeta Json had incorrect format!'
+const DV_VERSION_MISMATCH_MESSAGE = 'Unknown version of addressMeta! Expected "dv" to equal 1'
+const FLAT_WITHOUT_FLAT_TYPE_MESSAGE = 'Flat is specified, but flat type is not!'
```

Comment:
Why not use errors that are specfied in common domain?

Reply:
Because there's no rule specified on "How we throw server errors",
There was idea to throw error codes, but for now it's just a idea....so, I guess, it's resolved?